optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-client/src/main/resources/org/optaplanner/workbench/screens/domaineditor/client/resources/i18n/DomainEditorConstants.properties

```diff
@@ -46,4 +46,7 @@ DataObjectFieldPickerItemViewImpl.Descending=Descending
 DataObjectFieldPickerItemViewImpl.Reverse=Reverse
 DataObjectFieldPickerItemViewImpl.SortOrder=Sort order
 
-PlannerCheckTranslator.MultiplePlanningSolutionsToBeCreated=The operation will result in multiple Planning Solution instances within a single project
+PlannerCheckTranslator.MultiplePlanningSolutionsToBeCreated=The operation will result in multiple Planning Solution instances within a single project.
+PlannerCheckTranslator.ScoreHolderGlobalToBeDeleted=The operation will result in 'scoreHolder' global variable associated with this Planning Solution being deleted. This will cause build errors if the global is referenced in rules.
```

Comment:
We run a Maven [plugin](https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/blob/master/pom.xml#L923), "replacer", that replaces single quotes with two single quotes. Historically this has been for GWT, that needed to escape the single quote, however since you're using errai-ui ```TranslationService``` you don't need to worry about the escape.. So you either need exclude this properties file from the plugin, or don't use the single quote!